src/platforms/android/usage/advanced-usage.mdx

```diff
@@ -0,0 +1,303 @@
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        // Add the line below, the plugin that copies the binaries
+        // https://github.com/howardpang/androidNativeBundle/releases
+        classpath 'com.ydq.android.gradle.build.tool:nativeBundle:{version}'
+    }
+}
+```
+
+Once the dependency is declared, we can use it in the application build.gradle:
+
+```groovy
+apply plugin: 'com.ydq.android.gradle.native-aar.import'
+```
+
+The last step is to update the CmakeLists.txt configuration to link the Sentry NDK libraries:
+
+```
+# include paths generated by androidNativeBundle
+include (${ANDROID_GRADLE_NATIVE_BUNDLE_PLUGIN_MK})
+# change native-lib to your native lib's name
+target_link_libraries(native-lib ${ANDROID_GRADLE_NATIVE_MODULES})
+```
+
+Now you can use the Sentry NDK API just by including the sentry.h in your code:
+
+```c
+#include <jni.h>
+#include <android/log.h>
+#include <sentry.h>
+
+#define TAG "sentry-android-demo"
+extern "C" JNIEXPORT jstring JNICALL
+
+Java_io_sentry_demo_NativeDemo_crash(JNIEnv *env, jclass cls) {
+    __android_log_print(ANDROID_LOG_WARN, "", "Capture a message.");
+    sentry_value_t event = sentry_value_new_message_event(
+            /*   level */ SENTRY_LEVEL_INFO,
+            /*  logger */ "custom",
+            /* message */ "Sample message!"
+    );
+    sentry_capture_event(event);
+}
+```
+
+To symbolicate the stack trace from native code, we need to have access to the debug symbols of your application. Please check the full documentation on [uploading files](/workflow/debug-files/#uploading-files) to learn more about the upload of the debug symbols.
+
+Example of uploading all your .so files:
+
+```bash
+sentry-cli login
+sentry-cli upload-dif -o {YOUR ORGANISATION} -p {PROJECT} build/intermediates/merged_native_libs/{buildVariant}
+```
+
+The Sentry Gradle Plugin can be [configured](#gradle-configuration) to upload the debug symbols and sources automatically.
+
+### Crash Reporting
+
+Your application will crash whenever a thrown exception goes uncaught. The Sentry SDK catches the exception right before the crash and builds a crash report that will persist to the disk. The SDK will try to send the report right after the crash, but since the environment may be unstable at the crash time, the report is guaranteed to send once the application is started again.
+
+If there is a fatal error in your native code, the process is similar. The crash report might send before the app crashes, but will for sure send on restart.
+
+The NDK is not only catching the unhandled exceptions but is also set as a signal handler to be able to react to the signals from OS. When the application is about to crash, an error report is created and saved to disk. The SDK will try to send the report right after the crash, but since the environment may be unstable at the crash time, the report is guaranteed to send once the application is started again.
+
+## Manual Instrumentation
```

Comment:
this is fine to stay here, our SDK inits automatically, they don't need to SentryAndroid.init(...) unless they want to.